### PR TITLE
fix(widget): localStorage null check

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport#425

Ref: #_[ticket-number]_

### Dependencies

- Upstream: _[this PR points to another PR or depends on its release]_
- Downstream: _[PRs that depend on this one, either point to this or can only be released after this one is released]_

### Description

_[Document your changes, give context for reviewers, add images/videos of UI changes]_

### Release Plan

_[Document your release plan to minimize downtime]_

- [ ] Upstream dependencies are met
- [ ] _[action 1]_
- [ ] _[action 2]_

### CODE REVIEW

File: connect-widget/app/src/shared/localStorage/custom-color.ts [hunk 1]
```diff
@@ -31,11 +31,5 @@ const validateColor = (paramColor: string | null) => {
...
 
 const getLocalStorageColor = (): string | null => {
-  const localStorageColor = localStorage.getItem(colorLocalStorageKey);
-
-  if (localStorageColor) {
-    return localStorageColor;
-  }
-
-  return null;
+  return localStorage ? localStorage.getItem(colorLocalStorageKey) : null;
```
Comment (Goncharo) [inline connect-widget/app/src/shared/localStorage/custom-color.ts:34]: Pro code 🤩

**Review by Goncharo — APPROVED** [review]
:shipit:

**Review by leite08 — APPROVED** [review]
LGTM!

Only comment is to either fill out the sections in the PR template or remove them, if they're not needed. :)

**Comment by RamilGaripov** [comment]
> LGTM!
> 
> Only comment is to either fill out the sections in the PR template or remove them, if they're not needed. :)

Thank you! Will keep that in mind for future PRs!

**Pull request merged**